crowbar_framework/app/models/service_object.rb

```diff
@@ -1271,6 +1271,8 @@ def apply_role(role, inst, in_queue, bootstrap = false)
     pre_cached_nodes = {}
 
     # Each batch is a list of nodes that can be done in parallel.
+    # only run remote chef-clent on a node once
+    batches.uniq!
```

Comment:
yes, correct. in this case it will not have any effect (maybe it even shouldn't).
I only had the case where the list looked like `[["node1"], ["node1"]]`, that's why I sent this patch